examples/readme.md

```diff
@@ -0,0 +1,39 @@
+## Convert TensorFlow model to Core ML model format
```

Comment:
now that you have moved this description in its own readme which is good, can you delete the descriptions from the top level readme?

Reply:
Sure!

Reply:
Done.